app/processing/components/get-scheme.js

```diff
@@ -1,7 +1,9 @@
-const getScheme = (year, frequency, agreementNumber) => {
+const { LONG_NAMES } = require('../../constants/scheme-names')
+
+const getScheme = (year, frequency, agreementNumber, sourceSystem) => {
```

Comment:
Is sourceSystem descriptive enough? It is unclear what it does on a quick glance, especially compared to the rest of the parameters

Reply:
The we inherited name sourceSystem from the source file we received from SitiAgri it is not something that we created. I agree with you that the name is not descriptive enough, hence our reason for changing it to something more descriptive in the getScheme function:

https://github.com/DEFRA/ffc-pay-statement-constructor/blob/254026_SFI-23-Modify-statement-constructor-output-scheme-short-name-to-be-variable/app/processing/components/get-scheme.js